puffin-imgui/src/ui.rs

```diff
@@ -675,7 +675,7 @@ impl ProfilerUi {
         ui.next_column();
         {
             let num_fit = (ui.content_region_avail()[0] / self.options.frame_width).floor();
-            let num_fit = (num_fit as usize).max(1).min(frames.slowest.len());
+            let num_fit = (num_fit as usize).clamp(1, frames.slowest.len());
```

Comment:
Not sure because I'm not sure what this does, but is the previous code correct? Looks like min and max might have been inverted, since setting a max of 1 basically allows 0 or 1 for a usize.

Reply:
In fact, `max(1)` will set an _upper bound_ of 1, which was the surprising part, because it limited the `num_fit` to the {0, 1} set. I assumed this was inverted, hence my code -- but I can just put it back to what it was before, since I'm not sure what's the expectation there.

Reply:
To be clear, I do understand perfectly fine how `clamp` / `max` / `min` work, I'm not sure what's expected of the surrounding code in that part I've changed; is it expected that it be clamped to [0; 1]? or should it be clamped to [1; frames.slowest.len()] as I've "fixed" here?

Reply:
wow sorry, I've been massively confused by the syntax here, and you're right. So my code is totally equivalent, that's good.

Reply:
Just put it back? Or was it part of a clippy suggestion?

Looks like it has the lower bounds of 0-1, and then does upper bounds of at least 1, thus indeed 0-1. That seems a bit wrong to me. I would assume this code tries to at least have one fit up to how many items can fit. So clamp looks valid. Perhaps you can run the puffin imgui viewer and see how this Ui behaves with this change?